x/cosmwasmpool/genesis_test.go

```diff
@@ -20,7 +20,7 @@ func (s *PoolModuleSuite) TestInitGenesis() {
 		expectedTotalLiquidity = expectedTotalLiquidity.Add(initalDefaultSupply...)
 	}
 
-	pools, err := s.App.CosmwasmPoolKeeper.GetPoolsWithWasmKeeper(s.Ctx)
```

Comment:
This method was removed in favor of utilizing GetPools to contain the keeper, and serializable method that does not contain the keeper